dev/TabView/TabViewItem.cpp

```diff
@@ -146,6 +151,16 @@ void TabViewItem::OnCloseButtonOverlayModeChanged(winrt::TabViewCloseButtonOverl
     UpdateCloseButton();
 }
 
+winrt::TabView TabViewItem::GetParentTabView()
+{
+    return m_parentTabView.get();
+}
+
+void TabViewItem::SetParentTabView(winrt::TabView const& tabView)
+{
+    m_parentTabView = winrt::make_weak(tabView);
```

Comment:
I think the fact that this is needed indicates that the components are architectured incorrectly.  The TabViewItem shouldn't need to know about the Parent TabView. Instead I think that it might have been better to thave selection status held on the TabViewItem, which can raise Selection changed events that the TabView can use to manage the single selection.  That being said I don't think we should change this now... maybe.

Reply:
That's a really good point. The question that comes to my mind now, is why the NavigationViewItem has an IsSelected property, while the TabViewItem does not. Both act more or less like a tab item, in both cases only a single item can be selected at a time. Was/is there a specific reasoning why those two diverge here?

Reply:
TabViewItem does have an IsSelected property which is inherited from SelectorItem

Reply:
Oh missed that. The "Is selected" property doesn't appear in the official TabViewItem documentation. Also the TabViewItem/TabView do not care about that property at all.

Was this by design? I think @StephenLPeters is right here, that is something we should update. Currently only TabView knows about selection in contrast to NavigationView, where the items also know their selection status.

Reply:
> Oh missed that. The "Is selected" property doesn't appear in the official TabViewItem documentation. Also the TabViewItem/TabView do not care about that property at all.

The TabViewItem.IsSelected property does work though. See this XAML:
```xml
<muxc:TabView>
    <muxc:TabView.TabItems>
        <muxc:TabViewItem Header="Tab 1"/>
        <muxc:TabViewItem Header="Tab 2"/>
        <muxc:TabViewItem Header="Tab 3" IsSelected="True"/>
        <muxc:TabViewItem Header="Tab 4"/>
    </muxc:TabView.TabItems>
</muxc:TabView>
```
This gives me the expected result:
![image](https://user-images.githubusercontent.com/1398851/87227520-e0f94080-c39b-11ea-907a-24fd876b6237.png)

Reply:
So is ListView doing selection for us here then?

Reply:
I would assume so, yes. Same as with TreeView when we are in TreeView.SelectionMode = Single.

Reply:
The incomplete WinUI documentation is an issue though. @ranjeshj @StephenLPeters Can the WinUI API documentation be updated to also include all the inherited APIs like the OS XAML documentation? I aleady had to open the OS XAML documentation quite a few times for WinUI components (like the NavigationView) because that documentation also lists all the inherited APIs.

Reply:
Since IsSelected works here, I removed the "GetParentTabView" pattern/code.

Reply:
@chigy and @YuliKl for the documentation question

Reply:
@chingucoding would it makes sense to move the pointer to the parent tab view onto the TabViewItemAutomationPeer since the TabViewItem itself doesn't need it?

Reply:
But how would we set that property then? After all, we don't know when the AutomationPeer for a TabViewItem will be created, so TVI needs to "save" it until it will be needed. And since AutomationPeer are more or less "fire and forget" anyway (meaning they get created and can be used only once and get recreated for the next usage), the TVI AutomationPeer would always have to ask a TVI for it's parent TabView.

Reply:
makes sense